go/store/nbs/archive_build.go

```diff
@@ -628,3 +782,63 @@ func (cr *ChunkRelations) Add(a, b hash.Hash) {
 		cr.manyToGroup[h] = &merged
 	}
 }
+
+// SimpleChunkSourceCache is a simple cache for chunks. For the purposes of building archives, we want to avoid storing
+// all chunks in memory.
+type SimpleChunkSourceCache struct {
```

Comment:
(minor) Looks like this type is only used in this package, so you could lower case it to avoid having it exported, if you didn't intend on other packages to be able to use it.